### PR TITLE
[Tech debt] Consolidate addresses

### PR DESCRIPTION
**Pull request by Mottie**
## Description

Address data (e.g. lists of countries & states) have been duplicated throughout the repository. This PR is meant to consolidate all this data and use one source of truth - The data in https://github.com/department-of-veterans-affairs/vets-json-schema/blob/master/dist/constants.json

## Testing done

Updated all existing unit tests; all passing

## Screenshots

N/A

## Acceptance criteria
- [x] Less code, duplicate outcomes

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/platform/forms/address/index.js [hunk 1]
```diff
@@ -1,353 +1,10 @@
...
-    { label: 'Virginia', value: 'VA' },
-    { label: 'Washington', value: 'WA' },
-    { label: 'West Virginia', value: 'WV' },
-    { label: 'Wisconsin', value: 'WI' },
-    { label: 'Wyoming', value: 'WY' },
-  ],
-};
+// states.USA_OTHER = states.USA.concat([{ label: 'Other', value: 'Other' }]).sort(
+//   (a, b) => a.label.localeCompare(b.label),
+// );
```
Comment (Mottie) [inline src/platform/forms/address/index.js:5]: I couldn't find where this code was being used, so I only commented it out for now.

**Comment by kelsonic** [comment]
Hey! You're awesome! Thank you so much for doing this!!! 🎉 📈 🎉

File: src/applications/yellow-ribbon/containers/SearchForm/index.jsx [hunk 2]
```diff
@@ -7,7 +7,7 @@ import map from 'lodash/map';
 import { connect } from 'react-redux';
 // Relative imports.
 import ErrorableCheckbox from '@department-of-veterans-affairs/formation-react/ErrorableCheckbox';
-import STATES from 'platform/static-data/STATES.json';
+import { states as STATES } from 'vets-json-schema/dist/constants.json';
```
Comment (kelsonic) [inline src/applications/yellow-ribbon/containers/SearchForm/index.jsx:10]: Thank you!! 🥇

**Comment by Mottie** [comment]
Not included is the duplicate data in a few forms; mostly to prevent merge conflicts - discussed with form code owners
- HLR schema - I will remove in a future PR
- Form 2346 - to be removed by code owner
- Form 686c-674 - to be removed by code owner
- Consider updating the country list in vets-json-schema to include other country codes used by the vets360 component - https://github.com/department-of-veterans-affairs/vets-website/blob/380b24b3457c7fef0c555db4465cc89f3520327d/src/platform/user/profile/vet360/constants/countries.json#L1297

**Comment by kelsonic** [comment]
Heads up that there's another PR that has custom states out there that will need consolidating! Cross-referencing so that you folks are in touch! 🙂 https://github.com/department-of-veterans-affairs/vets-website/pull/11999/files#r406813416 @bradl3yC @mr0sari0

**Review by ncksllvn — APPROVED** [review]
I can speak for Yellow Ribbon and the Vet360 (VA Profile) work, but LGTM! 👍 Awesome work

**Review by ncksllvn — APPROVED** [review]

**Review by cohnjesse — APPROVED** [review]
LGTM

**Review by micahchiang — APPROVED** [review]
wowowow 😍

**Review by ndsprinkle — APPROVED** [review]

**Review by brangi — APPROVED** [review]

**Review by erikphansen — APPROVED** [review]

**Review by LinnJS — APPROVED** [review]
Love to see all that red! This was a huge undertaking, huge thank you for the above and beyond work!

**Review by rianfowler — APPROVED** [review]
fyi @Clark-W-Griswold

**Pull request merged**